instack.answers.sample

```diff
@@ -113,3 +113,12 @@ UNDERCLOUD_CEILOMETER_PASSWORD=
 # Ceilometer snmpd password
 # If left unset, one will be automatically generated
 UNDERCLOUD_CEILOMETER_SNMPD_PASSWORD=
+
+### Swift password ###
+# Swift service password
+# If left unset, one will be automatically generated
+UNDERCLOUD_SWIFT_PASSWORD=
+
+### Swift hash suffix ###
+# A random string to be used as a salt when hashing to determine mappings in the ring.
+UNDERCLOUD_SWIFT_HASH=unset
```

Comment:
why default this to unset? why not leave it blank here and then generate a random value in instack-install-undercloud like we do for the passwords?